vulture_os/toolkit/api_parser/perception_point_x_ray/perception_point_x_ray.py

```diff
@@ -206,11 +213,13 @@ def format_log(log):
                 del log["sample"]["disclaimers"]
         return dumps(log)
 
+    def execute_scans(self):
+        # Get "SCAN" logs
 
-    def execute(self):
         # Warning : the fetched logs are ordered in ASC (no option is available in API doc)
-        for log_type in self.LOG_TYPES:
-            since = self.last_collected_timestamps.get(f"perception_point_x_ray_{log_type}") or (timezone.now() - timedelta(days=30))
+        for log_type in self.SCAN_LOG_TYPES:
+            since = self.last_collected_timestamps.get(f"perception_point_x_ray_scan_{log_type}") or (
```

Comment:
Careful with that change: currently running collectors will have their pointer reset after upgrade if no migration is done at the same time!